src/applications/vaos/new-appointment/redux/helpers/getReasonCode.js

```diff
@@ -21,8 +21,11 @@ export function getReasonCode({ data, isCC, isDS }) {
     visit => visit.id === data.visitType,
   ).map(visit => visit.vsGUI);
 
-  if (isCC && data.reasonAdditionalInfo) {
-    reasonText = data.reasonAdditionalInfo.slice(0, 250);
+  if (isCC) {
+    reasonText = data.reasonAdditionalInfo?.slice(0, 250);
+    return {
+      text: reasonText,
+    };
```

Comment:
Moved check for 'reasonAdditionalInfo' inside the if statement because we don't want to return the default which is for VA request appointments.

Reply:
This looks good!